go.mod

```diff
@@ -1,31 +1,54 @@
 module github.com/portworx/talisman
 
-go 1.13
+go 1.15
```

Comment:
The BVT and Smoke jobs don't exercise talisman. So this will need to be manually verified. If you have a latest 2.11 cluster, we can use the talisman image from this PR to cleanup and verify.